modules/oe_content_publication/config/install/core.base_field_override.node.oe_publication.title.yml

```diff
@@ -7,7 +7,7 @@ id: node.oe_publication.title
 field_name: title
 entity_type: node
 bundle: oe_publication
-label: Title
+label: Page title
```

Comment:
This is out of scope, if this is needed, please open a new issue and we will prioritize it with the PO.